src/main/java/org/terasology/launcher/settings/LauncherSettingsValidator.java

```diff
@@ -29,19 +29,37 @@
 public final class LauncherSettingsValidator {
     private static final Logger logger = LoggerFactory.getLogger(LauncherSettingsValidator.class);
 
+    private static String removeUnsupportedJvmParameters(final String currentParams) {
+        String params = currentParams;
+
+        final String[] deprecatedParams = {"-XX:\\+UseParNewGC", "-XX:\\+UseConcMarkSweepGC", "-XX:ParallelGCThreads=10"};
```

Comment:
I was on the fence about whether to include the threads one or not. It's not technically an error, but I think it's unlikely to be a good default.

I guess we can deprecate it since it's much more likely it got there from the old defaults instead of on purpose.